app/controllers/clients_controller.rb

```diff
@@ -44,6 +44,11 @@ def destroy
     redirect_to clients_url, make_notice
   end
 
+  def without_assignments
```

Comment:
Maybe rename to `need_accompanying`, so you would have nicer routes file (see also comment for `routes.rb`)